File: programmers/js/lev0/배열 만들기 5.js
function solution(intStrs, k, s, l) {
  // var answer = [];
  // intStrs.forEach((el) => {
  //     let num = parseInt(el.slice(s, s + l));
  //     if (num > k) {
  //         answer.push(num);
  //     }
  // });
  // return answer;
  return intStrs.map((m) => +m.slice(s, s + l)).filter((m) => m > k);
}

File: programmers/java/lev0/배열 만들기 5.java
import java.util.ArrayList;
import java.util.List;

class Solution {
    public int[] solution(String[] intStrs, int k, int s, int l) { 
        List<Integer> answer = new ArrayList<>();
        
        for (String str : intStrs) {
            // Integer n = Integer.valueOf(str.substring(s, s+l));
            int n = Integer.parseInt(str.substring(s, s+l));
            if(n> k){
                answer.add(n);
            }
        }
        // return answer.stream().mapToInt(Integer::intValue).toArray();
        return answer.stream().mapToInt(i->i).toArray();
    }
}
